quickstarts/mongodb-prometheus/config.yml

```diff
@@ -0,0 +1,66 @@
+  ## MongoDB Monitoring
+
+  MongoDB enables the unlimited virtual scaling of applications. Utilities like mongostat and mongotop offer immediate results but fail to provide insights into trends in a highly graphical visual dashboard. MongoDB dashboards provide insights into key metrics like RAM usage, operations per second, page fault, disk size, lock %, and app and database performance at a glance. 
+
+  Avoid slow queries with  proper indexes that impact performance. Instantly monitor your entire MongoDB database with our instant observability kit or the MongoDB free monitoring tool.  
+
+  The critical differences between MongoDB free monitoring and monitoring MongoDB with New Relic’s instant observability quickstart are efficiency, usability, scope, and cost. MongoDB free monitoring focuses on standalone instances and replica sets. Data collected on disk utilization, memory, and operation execution times are uploaded periodically. 
+
+
+  ### What’s Included?
+
+  New Relic + MongoDB quickstart - New Relic’s instant observability quickstart provides multiple monitoring parameters like operations per second, transactions, and queries by default.
+
+  New Relic’s MongoDB quickstart contains multiple dashboards, including:
+  - Total Commands, failed commands per second, bytes in & out per second, available connections, and more. 
+
+  Monitor MongoDB with New Relic to quickly gain improved distribution and increased visibility into real-time user and app response times, throughput and breakdown by component and layer, and long-term data trends over time.  
+
+  ### Value of MongoDB Quickstarts
+
+  [MongoDB performance monitoring with New Relic](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/mongodb-monitoring-integration-new/) offers advanced features, including:
+  - Obtain app performance insights.
+  - Create custom queries and charts of your data integrations.
+  - Filter and analyze configuration data and metrics in Infrastructure UI.
+
+  New Relic’s instant observability quickstart helps developers reduce administrative overheads and accelerate time to value. As New Relic is SaaS-based, you also don’t have to worry about maintenance or onboarding.
+
+summary: |
+  New Relic's MongoDB quickstart provides multiple customized dashboards including total commands, requests per second, and database size. Install the quickstart to better understand utilization of resources and monitor performance issues.
+
+# Support level
+level: New Relic
+
+# Design
+icon: logo.jpeg
+website: https://www.mongodb.com/
+
+# Authors of the quickstart
+authors:
+  - New Relic
+
+documentation:
+  - name: MongoDB
+    description: |
+      Open source, document-oriented database where you store data in JSON-like files with dynamic schemas.
+    url: https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/mongodb-monitoring-integration-new/
+
+installPlans:
+  - mongodb-integration-docs
```

Comment:
you are missing the install plan https://github.com/newrelic-forks/newrelic-quickstarts/blob/2af00bf2179f3fad9a812cbe01fc6ece39275357/install/on-host-integration/winservices/install.yml

Reply:
Fixed. Thx